src/agent/multitype_queue/src/persistence_factory.hpp

```diff
@@ -7,8 +7,13 @@
 #include <string>
```

Comment:
The `string` header is no longer needed here, but could be needed in the source file...